### PR TITLE
Permission checks and user elevation

### PR DESCRIPTION
**Pull request by LegenJCdary**

### CODE REVIEW

File: ansible_deployer/command_line.py [hunk 1]
```diff
@@ -513,18 +561,16 @@ def main():
 
     start_ts = datetime.datetime.now().strftime("%Y%m%d_%H%M%S")
 
-    if len(sys.argv) < 2:
+    if len(sys.argv) < 3:
```
Comment (cinek810) [inline ansible_deployer/command_line.py:564]: This is wrong and is the reason why CI fails today. circle CI isn't related with the publication it calls pip on a local checkout

Comment (cinek810) [reply ansible_deployer/command_line.py:564]: think about ansible-deployer list (it's only two and it's fine)

**Review by cinek810 — CHANGES_REQUESTED** [review]

File: ansible_deployer/command_line.py (hunk 1 above)
Comment (LegenJCdary) [reply ansible_deployer/command_line.py:564]: Yes, we need version 0.0.17. Commit is updated. Please upload the package I sent you on teams, credentials are required.

**Comment by cinek810** [comment]
Well.. it's definitively to big PR. I'm not opposed to move logger creation earlier, but.. if we are moving it earlier the logs that were just stderr print should be changed to logger at the same time.

**Review by cinek810 — CHANGES_REQUESTED** [review]

**Comment by LegenJCdary** [comment]
Commits:
f98e655 Move logger creation earlier
f6bc8b5 Add checking permissions and owner for cfg file
76b8859 Add verification and pulling of git commit (no tests)
moved to #55

**Pull request closed without merging**